src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Connections.cs

```diff
@@ -287,24 +287,36 @@ private static IPAddress ParseIPv4HexString(string hexAddress)
         }
 
         // Parses a 128-bit IPv6 Address stored as 4 concatenated 32-bit hex numbers.
+        // If isSequence is true it assumes that hexAddress is in sequence of IPv6 bytes.
         // First number corresponds to lower address part
         // E.g. IP-address:                           fe80::215:5dff:fe00:402
         //      It's bytes in direct order:           FE-80-00-00  00-00-00-00  02-15-5D-FF  FE-00-04-02
         //      It's represenation in /proc/net/tcp6: 00-00-80-FE  00-00-00-00  FF-5D-15-02  02-04-00-FE
         //                                             (dashes and spaces added above for readability)
         // Strings passed to this must be 32 characters in length.
-        private static IPAddress ParseIPv6HexString(string hexAddress)
+        private static IPAddress ParseIPv6HexString(string hexAddress, bool isSequence = false)
```

Comment:
nit: alternative name for isSequence -> isNetworkOrder

Reply:
updated.